### DISCUSSION TITLE
Some Issues with JKBMS

### DISCUSSION THREAD

**Discussion by fast-quality** [Louisvdw/dbus-serialbattery#333] [Q&A] [answered]
Hello Louis and Community,

at first I must say that I´m very glad by using this platform.
I´m new in this and have learned a lot in the last days.
Thanks to all and especially to Louis for his great work !

To the Problems, I searched for it, but i doesn´t find the right information.
My System is Online since two days and fortunately it seems to be that everything work.

1.) I can see the SOC in the Device List of the Remote Console and on VRM in Extended View.
But in the Main Page of the Remote Console, there is no Information to this shown in the battery icon.
It is because my system is to new, or something, I don´t understand it.

2.) When I changed something in the "utils.py", it is always necessary to reboot the system ?

3.) I have configured the MP2 for ESS and make some configes in the assistant. Like choose all 
![ess](https://user-images.githubusercontent.com/120103744/206784128-39a83a7d-3df8-4152-ae19-f64ae85b5a8c.PNG)
In the morning the batterie are going to a lower level, I get a warning message from the Multi on the Remote Console.
That I fall below the set voltage of 48V. I think that when in Venus DVCC is on the Settings in the MP2 are not active ?!
Therefore I think I don´t have to get such messages. 

I will be nice when somebody can help me.

Greetings from Germany
Martin

**Comment by fast-quality** [comment]
Hello, i solved the first problem. I find the Settings in the Venus OS, all fine, my mistake.

Another question, when I set the name of the jk it will not save correctly, after reboot something, it's the default one

**Comment by Louisvdw** [comment, marked as answer]
Hi Martin

When you make changes in utils.py, then yes you do need to reboot or else the changes will not apply.

The low voltage alarm at 48V will be coming from your BMS. When you set up your BMS the low voltage value for your battery should be less than the values in utils.py, so if you use the default 2.9V for the min cell voltage and you have a 15 cell battery then your BMS low voltage alarm should be less than that (2.9V x 15 = 43.5V). If not then the BMS will raise the low voltage alarm and the driver will publish that to the victron system.

The custom name that is not remembered is a known issue and will be changed in a future release.

Regards Louis

**Comment by fast-quality** [comment]
Hey thanks for your information,I understand the voltage settings and set it, I think, in a right way. It is possible to check the settings, the bms driver set in the mppt control. (DVCC on). I had some strange behavior after updating and downgrading your driver. I had done this, because of some problems with another skill (dbus-Shelly..).When I install serialbatttery the Shelly driver faults. Otherwise it works.Regards Martin

**Reply by Louisvdw** [reply]
The CCL,DCL,CVL that the driver sets can be seen under the Parameters tab.
